moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -62,11 +62,10 @@ const useStyles = makeStyles((theme) => ({
  * @param {string} title - The title header of the component
```

Comment:
Another tiny one - looks like the title prop was removed from this component and it stayed in the docstring

Reply:
🙏 resolved in #873